samples/instanceadmin/instanceadmin.py

```diff
@@ -1,259 +1,270 @@
-#!/usr/bin/env python

```

Comment:
I ran `black` on this file. I'm not sure what change it made that makes the entire file different, but that's what happened. I could unblacken, but there are some manual edits I did before running `black` that I'd have to reproduce.